burotel/indico_burotel/template_overrides/core/rb/emails/reservations/reminders/finishing_bookings.html

```diff
@@ -0,0 +1,18 @@
+{% extends '~rb/emails/reservations/reminders/finishing_bookings.html' %}
+
+{% block subject %}Some of your desk reservations are finishing soon{% endblock %}
+
+{% block body -%}
+    {%- if reservations | length == 1 -%}
+        There is one desk reservation under your name that will soon come to an end:<br>
+    {%- else -%}
+        There are desk reservations under your name that will soon come to an end:<br>
+    {%- endif %}
+    <ul>
```

Comment:
i think this is duplicated from the original template. can we move it into a macro to avoid repeating it? or can we put the text above in a block in the core template and just override that block here?

Reply:
Yeah, I was gonna do this anyway. But thanks for the comment ;)